app/views/items/show.html.erb

```diff
@@ -14,7 +14,7 @@
         No current bids
       </p>
       <% else %>
-        Highest Bid: <%= @item.high_bid %><br>
+        Highest Bid: <%= number_to_currency(@item.high_bid) %><br>
```

Comment:
The number_to_currency helper is Built into Rails!  It's a method in the ActionView class, which is a class available to views in Rails:

http://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html